### PR TITLE
0.19

### PR DESCRIPTION
**Pull request by MoOx**
wip

### CODE REVIEW

File: themes/phenomic-theme-base/content/about.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+For now [Phenomic does not provide a native way to extend themes](https://github.com/MoOx/phenomic/issues/508) but since it's based on
+React and use a component approach, it's pretty easy to make your own collection
+of layouts and components and reuse those.
+
+---
+
+This website is made with [Phenomic](https://phenomic.io/).
+
+If you are curious about it, you can take a look to
```
Comment (ben-eb) [inline themes/phenomic-theme-base/content/about.md]: *a look at*

File: themes/phenomic-theme-base/content/index.md [hunk 2]
```diff
@@ -1,32 +1,43 @@
...
+  <a href="https://facebook.github.io/react/">
+    <img alt="" src="assets/react.svg" width="16" height="16" />
+    React
+  </a>.
+</small>
 
 ---
 
-You may want to take a look to the <a href="/404.html">404.html</a>.
+You may want to take a look to the [404.html](/404.html) and tweak the render.
```
Comment (ben-eb) [inline themes/phenomic-theme-base/content/index.md]: *a look at*

File: themes/phenomic-theme-base/content/index.md [hunk 3]
```diff
@@ -1,32 +1,43 @@
...
+
+If you want to adjust page layouts, directly adjust them in ``src/layouts/``.  
+If you want to adjust small parts like header, footer, buttons, etc, you will
+find those in ``src/components/``.  
+For colors and common CSS variables, you will find stuff to adjust in ``postcss.plugins.js``.
```
Comment (ben-eb) [inline themes/phenomic-theme-base/content/index.md]: I think there's too many adjusts here, it doesn't read all that well. Here's an alternate version:

> If you need to make adjustments, you can directly alter page layouts in ``src/layouts/``; small things like the header, footer, buttons and so on can be changed in ``src/components/``; and for colors and common CSS variables, you can modify ``postcss.plugins.js``.

File: themes/phenomic-theme-base/content/about.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+  label: Discover Phenomic now
+  link: "https://phenomic.io/"
+  props:
+    big: true
+---
+
+You are currently viewing Phenomic base theme.
+
+For now [Phenomic does not provide a native way to extend themes](https://github.com/MoOx/phenomic/issues/508) but since it's based on
+React and use a component approach, it's pretty easy to make your own collection
```
Comment (ben-eb) [inline themes/phenomic-theme-base/content/about.md]: > ...uses a component approach, it's pretty easy to make your own reusable collection of layouts and components.

File: docs/content/docs/usage/styling.md [hunk 5]
```diff
@@ -72,12 +80,75 @@ whenever you can.
...
+[Glamor](https://github.com/threepointone/glamor/),
+[styled-components](https://styled-components.com/)
+and
+[Aphrodite](https://github.com/Khan/aphrodite)
+should work out of the box.
+Static rendering step will take that into account and will pre-render styles
+for you. Nothing to setup.
+It’s even injecting all requirements if you want to rehydrate on startup (in ``window._glamor`` or ``window._aphrodite``).
+
+An optional (but recommended) step is to enable re-hydratation.
```
Comment (ben-eb) [inline docs/content/docs/usage/styling.md]: *re-hydration*

File: docs/content/docs/usage/styling.md [hunk 6]
```diff
@@ -72,12 +80,75 @@ whenever you can.
...
+[See aphrodite server documentation](https://github.com/Khan/aphrodite#server-side-rendering)
+
+#### A note on pure inline styles
 
-Unfortunately, inline styles don't play well with pre-rendering for now. When
-we build the static version, we don't know where the site will be viewed on, so
-viewport adjustments can't be done properly and will therefore result in some
-visual changes/re-rendering.
+Unfortunately, pure inline styles don't play well with pre-rendering for now.
+When we build the static version, we don't know where the site will be viewed on,
```
Comment (ben-eb) [inline docs/content/docs/usage/styling.md]: *where* => *what devices*

File: docs/content/index.md [hunk 7]
```diff
@@ -36,27 +38,15 @@ the minimal amount of data for each new page**
 (a single file that only contains your page data).
 
 <center style="font-weight: bold"><a href="showcase/">
-  Take a look at websites using Phenomic 👀
+  See how is using Phenomic
```
Comment (ben-eb) [inline docs/content/index.md]: *See what it's like to use Phenomic*

Comment (MoOx) [inline docs/content/index.md]: What I want to write was "See WHO is using Phenomic" :)

Comment (ben-eb) [inline docs/content/index.md]: Ahh OK. Crisis averted! 👍

**Pull request merged**